processing/src/main/java/org/apache/carbondata/processing/loading/parser/impl/RowParserImpl.java

```diff
@@ -109,7 +109,9 @@ public RowParserImpl(DataField[] output, CarbonDataLoadConfiguration configurati
       return new String[numberOfColumns];
     }
     // If number of columns are less in a row then create new array with same size of header.
```

Comment:
please update the comment according to new conditon

Reply:
done